app/models/post.rb

```diff
@@ -4,7 +4,7 @@ class Post < ApplicationRecord
   has_many :likes
 
   def update_posts_counter
-    update(posts_counter: comments.count)
+    update(posts_counter: post.count)
```

Comment:
[REQUIRED] Kindly fix this code: you have `post.count` when it should be `posts.count`